app/controllers/match_list_base_controller.rb

```diff
@@ -168,6 +168,16 @@ def available_programs
   end
   helper_method :available_programs
 
+  def available_contact_types
+    MatchContacts.input_names
```

Comment:
Is this the right set?

Reply:
I believe so, though since you are only using it once, I'd probably not bother with the controller method and helper method, just shove `MatchContacts.input_names` in the view.